src/framex/driver/cache.py

```
@@ -0,0 +1,311 @@
+            logger.warning(f"Failed to prepare request cache for {request.method} {path}: {exc}")
+            response.headers[CACHE_STATUS_HEADER] = CacheStatus.BYPASS
+            return await invoke()
+
+        action = _cache_action(request)
+        if action == CacheAction.USE:
+            try:
+                cached_value = await self.get(store_key)
+                if cached_value is not None:
+                    response.headers[CACHE_STATUS_HEADER] = CacheStatus.HIT
+                    return cached_value
+            except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)